frontend/src/Components/RoadDetails/ImageClick.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+interface ImageClickProps {
+  imageUrl: string;
+  onClose: () => void;
+  onNavigate: (direction: number) => void;
+  currentImageIndex: number;
+  totalImages: number;
+}
+
+const ImageClick: React.FC<ImageClickProps> = ({ imageUrl, onClose }) => {
+  return (
+    <div className="image-click-modal">
```

Comment:
```suggestion
    <div className="image-click-modal" onClick={onClose}>
```
So the user can close the window by clicking outside the image